src/app.css

```diff
@@ -0,0 +1,205 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+#mainNav {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 30px;
+
+  /* background-color: grey; */
+  align-items: center;
+  align-content: center;
+}
+
+#ham {
+  width: 24px;
+  height: 24px;
+  cursor: pointer;
+}
+
+#logo {
+  width: 124px;
+  height: auto;
+  cursor: pointer;
+}
+
+#search-area {
+  display: flex;
+  justify-content: space-between;
+}
+
+#search-area form {
+  display: flex;
+  align-items: center;
+  position: relative;
+}
+
+#search-area input {
+  width: 563px;
+  height: 20px;
+  padding: 10px;
+  font-size: 20px;
+}
+
+#search {
+  height: 100%;
+  width: 64px;
+
+  /* background-color: lightgrey; */
+  border: none;
+  position: absolute;
+  right: 0;
+}
+
+#search-area #mic {
+  border-radius: 50%;
+  height: 44px;
+  width: 44px;
+  border: none;
+  margin-left: 10px;
+  cursor: pointer;
+}
+
+#left {
+  display: flex;
+  align-items: center;
+  align-content: center;
+}
+
+#left #ham {
+  padding-left: 10px;
+  padding-top: 10px;
+  margin-right: 15px;
+}
+
+#right {
+  display: flex;
+  width: 150px;
+  align-items: center;
+  align-content: center;
+  justify-content: space-between;
+}
+
+#right i {
+  cursor: pointer;
+}
+
+.likes-container span {
+  padding-right: 15px;
+  cursor: pointer;
+}
+
+.all-row span {
+  padding: 3px 7px;
+  font-size: 18px;
+  border: 1px solid black;
+  border-radius: 20px;
+  cursor: pointer;
+}
+
+#right span {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border: 1px solid black;
+  width: 32px;
+  height: 32px;
+  border-radius: 50%;
+  background-color: palegreen;
+  cursor: pointer;
+}
+
+/* #right span {
+  justify-content: center;
+  display: flex;
+  border: 1px solid black;
+  width: 25px;
+  height: 32px;
+  border-radius: 40%;
+  background-color: green;
+  cursor: pointer;
+} */
+
```

Comment:
- [ ] I suggest that you try not to leave commented pieces of code in your codebase as this clutters your codebase and makes it less readable. Find more in the microverse HTML and CSS [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)